core-services/image-mirroring/tekton/mapping_tekton_v0_14_quay

```diff
@@ -0,0 +1,10 @@
+registry.svc.ci.openshift.org/openshift/tektoncd-v0.14.1:tektoncd-pipeline-controller quay.io/openshift-pipeline/tektoncd-pipeline-controller:v0.14.1
+registry.svc.ci.openshift.org/openshift/tektoncd-v0.14.1:tektoncd-pipeline-creds-init quay.io/openshift-pipeline/tektoncd-pipeline-creds-init:v0.14.1
+registry.svc.ci.openshift.org/openshift/tektoncd-v0.14.1:tektoncd-pipeline-entrypoint quay.io/openshift-pipeline/tektoncd-pipeline-entrypoint:v0.14.1
+registry.svc.ci.openshift.org/openshift/tektoncd-v0.14.1:tektoncd-pipeline-nop quay.io/openshift-pipeline/tektoncd-pipeline-nop:v0.14.1
+registry.svc.ci.openshift.org/openshift/tektoncd-v0.14.1:tektoncd-pipeline-git-init quay.io/openshift-pipeline/tektoncd-pipeline-git-init:v0.14.1
+registry.svc.ci.openshift.org/openshift/tektoncd-v0.14.1:tektoncd-pipeline-imagedigestexporter quay.io/openshift-pipeline/tektoncd-pipeline-imagedigestexporter:v0.14.1
+registry.svc.ci.openshift.org/openshift/tektoncd-v0.14.1:tektoncd-pipeline-kubeconfigwriter quay.io/openshift-pipeline/tektoncd-pipeline-kubeconfigwriter:v0.14.1
+registry.svc.ci.openshift.org/openshift/tektoncd-v0.14.1:tektoncd-pipeline-webhook quay.io/openshift-pipeline/tektoncd-pipeline-webhook:v0.14.1
+registry.svc.ci.openshift.org/openshift/tektoncd-v0.14.1:tektoncd-pipeline-pullrequest-init quay.io/openshift-pipeline/tektoncd-pipeline-pullrequest-init:v0.14.1
+registry.svc.ci.openshift.org/openshift/tektoncd-v0.14.1:tektoncd-pipeline-gcs-fetcher quay.io/openshift-pipeline/tektoncd-pipeline-gcs-fetcher:v0.14.1
```

Comment:
hi, the image mirroring should be from be from `tektoncd-v0.14.1-4.3` (or `tektoncd-v0.14.1-4.4` or `tektoncd-v0.14.1-4.5`) to `:v0.14.1` . isn't it

Reply:
fixed it